src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -127,7 +127,7 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     }
     const eventReference = {...event};
     if (eventReference.id) {
-      delete eventReference.id;
+      delete (eventReference as any).id;
```

Comment:
`eventReference` should be of type Event. I think the dev did this `const eventReference = {...event}; ` because he didn't want to mutate the `event` state.